tensorflow/compiler/tf2xla/kernels/split_op.cc

```diff
@@ -79,12 +79,14 @@ class SplitOp : public XlaOpKernel {
     // split dimensions varies for each output.
     std::vector<int64> begin;
     std::vector<int64> limits;
+    std::vector<int64> strides;
```

Comment:
indeed. i was maintaining coding style, but agreed it would be better to be more efficient.